### ISSUE TITLE
"Last updated on" part updated without change in content.

### ISSUE THREAD

**Issue by hermanp** [hermanp/website#7]
"Last updated on" part updated without change in content on the Resources menu. Perhaps this is related to enableGitInfo in config.toml? This should be only updated when the content changes on the specific site.

**Comment by hermanp** [comment]
After set `enableGitInfo = true`, the problem still occur. For example, the post [How I created this blogdown site](https://hermanp.github.io/en/post/how-this-blogdown-site-was-created/) was last updated on 2020-11-25, at the time of writing this comment. Yet, the page displays that it was updated on 2020-11-30, the last time a commit was pushed to the GitHub Pages site's repo. Visual evidence:
![Wrong update date](https://user-images.githubusercontent.com/16311451/100598183-ac603400-32fe-11eb-959c-c62022d3e821.png)
